examples/cockroachdb/cockroachdb-statefulset.yaml

```diff
@@ -66,6 +78,21 @@ spec:
       labels:
         app: cockroachdb
       annotations:
+        scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
Thanks, I've opened https://github.com/kubernetes/kubernetes/issues/39310

Also, I've switched to `preferredDuringSchedulingIgnoredDuringExecution`, since it won't break anything to have multiple instances on the same node, it's just strongly preferred not to.